src/styles/styles.js

```diff
@@ -526,6 +526,7 @@ const styles = {
     },
     expensiTextInputDesktop: {
         outline: 'none',
+        boxShadow: 'none',
```

Comment:
This will throw up in Native . Please use `addOutlineWidth(0)`

Reply:
Changed